Source/SIMPLib/CoreFilters/MultiThresholdObjects.cpp

```diff
@@ -213,13 +209,12 @@ void MultiThresholdObjects::execute()
   {
     ThresholdFilterHelper filter(static_cast<SIMPL::Comparison::Enumeration>(comp_0.compOperator), comp_0.compValue, m_DestinationPtr.lock().get());
     // Run the first threshold and store the results in our output array
-    err = filter.execute(m->getAttributeMatrix(amName)->getAttributeArray(comp_0.attributeArrayName).get(), m_DestinationPtr.lock().get());
+    int err = filter.execute(m->getAttributeMatrix(amName)->getAttributeArray(comp_0.attributeArrayName).get(), m_DestinationPtr.lock().get());
```

Comment:
This isn't the first occurrence, but you probably shouldn't replace `int32_t` with `int`, even if they technically mean the same thing.  int32_t declares a stronger intent and the type is intentionally 32-bits instead of the default int type.  There are plenty of places where we use the default int type because of Qt where int does not make any sense (i.e. this array size is -5), so declaring this as int32_t tells the reader that this is intentional and it actually should be the type we say it is.  I'm not so worried about the places where it wasn't int32_t, but erasing the intent should be avoided where possible.